src/main/io/serial_cli.c

```diff
@@ -1410,6 +1410,7 @@ static void cliReboot(void) {
     cliPrint("\r\nRebooting");
     waitForSerialPortToFinishTransmitting(cliPort);
     stopMotors();
+    handleOneshotFeatureChangeOnRestart();
```

Comment:
Some delay should be inserted between `stopMotors()` and 'pwmShutdownPulsesForAllMotors()'. Few stop pulses should be generated before disabling PWM completely.